lib/logger.ts

```
@@ -0,0 +1,130 @@
+
+      // Log entry for file
+      const logEntry = {
+        timestamp,
+        level,
+        message,
+        ...(data ? { data } : {}),
+      };
+
+      // Write to log file
+      if (this.logStream) {
+        this.logStream.write(JSON.stringify(logEntry) + "\n");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)